Sources/NIOCore/EventLoop.swift

```diff
@@ -21,18 +21,13 @@ import Dispatch
 /// will be notified once the execution is complete.
 public struct Scheduled<T> {
     /* private but usableFromInline */ @usableFromInline let _promise: EventLoopPromise<T>
+    
+    @usableFromInline let cancellationTask: (() -> Void)
```

Comment:
Presumably this would be `private` if it wasn't `@usableFromInline`. By convention in these cases we prefix the name with a `_` so this should be `_cancellationTask`.